### PR TITLE
Add keysight manifest for deploy

### PR DESCRIPTION
**Pull request by marcushines**

### CODE REVIEW

**Review by alexmasi — COMMENTED** [review]
the versions file should be named ixia-configmap.yaml: https://github.com/openconfig/kne/blob/1b4d2060101e3df472665f651c383efd322e0264/deploy/deploy.go#L826

but I like this pattern better, can we change it internally as well during the sync?

**Comment by frasieroh** [comment]
can we temporarily accept both `ixia-configmap` and `ixia-version` and have them be equivalent and mutually exclusive? We have an internal repo that distributes our own manifests and images to keeps things stable, and I'd prefer not to have it only work with versions of KNE before or after this change.

File: manifests/keysight/ixiatg-version.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
```
Comment (alexmasi) [inline manifests/keysight/ixiatg-version.yaml:1]: rename this file to ixiatg-configmap

File: deploy/kne/test.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (alexmasi) [inline deploy/kne/test.yaml:1]: this file looks leftover from manual testing?

**Comment by alexmasi** [comment]
the presubmit failed because this PR does not have the changes on the main branch that fix the issue

**Review by alexmasi — DISMISSED** [review]

**Review by alexmasi — DISMISSED** [review]

**Review by alexmasi — APPROVED** [review]

**Pull request merged**